scripts/zones/Chocobo_Circuit/npcs/Couzanne.lua

```diff
@@ -0,0 +1,27 @@
+-----------------------------------
+-- Area: Chocobo_Circuit
+-- NPC: Couzanne
+-- !pos -107.947 -14.500 -133.451 70
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    player:startEvent(332)
+end
+
+entity.onEventUpdate = function(player, csid, option)
+end
+
+entity.onEventFinsih = function(player, csid, option)
+    if
+        csid == 332 and
+        option == 1
+    then
+        player:setPos(-387.2634, -5.000, -490.0573) -- exits to Biorie
+    end
+end
```

Comment:
For these exit NPCs you shouldn't be setting pos or doing anything in the onEventFinish. Instead when you start the cutscene you should add whatever the corresponding parameter is for the entry NPC. For example with Couzanne you would do `player:startEvent(332, 5)` because Biorie uses 5 and these 2 NPCs are linked. Likewise Azainnie would use `player:startEvent(328, 1)` because Ilsoire uses 1 etc.

Reply:
I would double check that you have nothing in the onEventFinish and that the startEvent parameters are correct. I've tested and confirmed that `player:startEvent(csid, entry)` is the correct format for the exit NPCs and takes you to the correct place depending on the entry parameter. Note that the 1-5 index comes immediately after the csid unlike the entry NPCs. Example:
```lua
-----------------------------------
-- Area: Chocobo_Circuit
-- NPC: Perdric
-- !pos -84.007 -14.500 -133.451 70
-----------------------------------
local entity = {}

entity.onTrade = function(player, npc, trade)
end

entity.onTrigger = function(player, npc)
    player:startEvent(330, 3)
end

entity.onEventUpdate = function(player, csid, option)
end

entity.onEventFinsih = function(player, csid, option)
end

return entity
```